paddlets/datasets/splitter.py

```diff
@@ -29,13 +29,13 @@ class SplitterBase(metaclass=ABCMeta):
     """
 
     def __init__(self,
-                 skip_size: int = 0,
-                 verbose: bool = True) -> None:
+                 skip_size: Optional[int] = 0,
+                 verbose: Optional[bool] = True) -> None:
         self._skip_size = skip_size
         self._verbose = verbose
 
     def split(self, dataset: TSDataset,
-              return_index: bool = False) -> Union[TSDataset, pd.DatetimeIndex, pd.RangeIndex]:
+              return_index: Optional[bool] = False) -> Union[TSDataset, pd.DatetimeIndex, pd.RangeIndex]:
```

Comment:
should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional